index.html

```diff
@@ -0,0 +1,139 @@
+       the beautiful natural surroundings of Seoul’s landmark Hangang River.</p> 
+    </div>
+    <p class="date">2022 juin 25 (Sat)-11 july (Mon)</p>
+    <p class="place">@Bop Club and Bar and More</p>
+  </div>
+  </section>
+
+                            <!--Program section-->
+  <section class="program">
+  <h2 class="titre">Main program</h2>
+  <hr>
+  <div class="cards">
+    <div class='card'>  
+      <div class="prgimg">
+        <i class="fa-solid fa-microphone fa-2xl prgimg"></i>
+      </div>
+      <h3>Open entry</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-champagne-glasses fa-2xl prgimg"></i>
+      </div>
+      <h3>Boom Party</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-burger fa-2xl prgimg"></i>
+     </div>
+      <h3>Buffet Dinner</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-compact-disc fa-2xl prgimg"></i>
+      </div>
+      <h3>pink party</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+    <div class='card'>
+      <div class="prgimg">
+        <i class="fa-solid fa-guitar fa-2xl prgimg"></i>
+    </div>
+      <h3>Black Band</h3>
+      <p class="text">Authoritatively initiate worldwide services through premier best practices.</p>
+    </div>
+  </div>
+    <button class="programButton">Reserve Seat Now</button>
+  </section>
+  
+                         <!--Dj section-->
+  <section class="dj">
+       <h2  class="titre">Featured Djs</h2>
+       <hr>
+       <div class='djitems'>
+       </div>
+  </section> 
+                            <!--Sponsor section-->
+    <section class="Sponsor">
+      <h3>Partners</h3>
+      <hr>
+    <ul class="sponsorList">
+      <li class="design sponsorImg"><img src="images/cola.jpg" alt="cola"></li>
+      <li class="design sponsorImg"><img src="images/canon.png" alt="canon"></li>
+      <li class="design sponsorImg"><img src="images/pepsi.png" alt="pepsi"></li>
+      <li class="design sponsorImg"><img src="images/samsung.png" alt="samsung"></li>
+    </ul>
+    </section>
+     
+                             <!--Footer Section-->
+    <section class="footer">
+      <div class="footerlogo">
+        <img  src="images/logoheader.png" alt="Tech Crunch logo">
+    </div>
+    <div class="footerText">
+     <strong> 2022 E-ticket. Some rights reserved.</strong><br>E-Ticket in Association with Toyota.
+    </div>
+    </section>
```

Comment:
- [ ] Please replace `<section>` tag with `<footer>` tag. Let's follow the HTML best practices and it will also make sure that the code is semantic.